### ISSUE TITLE
Selectable TreeView: subitems are selected is announced multiple times

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2444]
**Describe the bug**
a11y Justin reported that selectable treeviews have a repeated announcement:
https://github.com/user-attachments/assets/cd97da71-5635-4d7e-8e3d-ed6d2011a138

I can reproduce this in our docs as well with VO.

**To Reproduce**
1. Go to a selectable tree view and expand a parent item
2. Use a screenreader (can reproduce with NVDA + VO) and listen to "All subitems are selected" getting announced the number of parent children inside the selected item

**Expected behavior**
Unsure. Probably should not announce repeatedly, but we need to investigate other Trees and determine how it should behave

**Comment by serhiiyanhol-cengage** [comment]
### macOS 
**chrome**: the message is announced twice, also as collapsed/expanded
<img width="1498" height="1169" alt="Image" src="https://github.com/user-attachments/assets/b4a3c65d-5e41-42ec-bb14-b51cf695ce2f" />
https://github.com/user-attachments/assets/5f2842e5-88f2-4391-9a4a-68a25f4427b0
**safari**: selected not announced
https://github.com/user-attachments/assets/7b2bbabc-8e25-45ba-8086-b43d72877ad4
**firefox**: selected not announced, 3 times collapsed/expanded
https://github.com/user-attachments/assets/879a60f3-d0f6-41f6-8cb5-22b2a9bb63a2

### windows
**chrome**: looks like ok
<img width="908" height="1204" alt="Image" src="https://github.com/user-attachments/assets/6bba808d-3d02-4cfe-8435-1c436de812ec" />
**firefox**: twice selected/unselected announce
<img width="1265" height="1205" alt="Image" src="https://github.com/user-attachments/assets/93a3db96-1724-4e7a-a5a3-91846d17a566" />
